--- ref
+++ di
@@ -52,8 +52,6 @@
 ----------------
 
 Block 1, Block Map Extension Block:
-  Block   2: CTA-861 Extension Block
-  Block   3: CTA-861 Extension Block
 Checksum: 0x0c
 
 ----------------
@@ -88,21 +86,6 @@
       Max channels: 2
       Supported sample rates (kHz): 48 44.1 32
       Supported sample sizes (bits): 24 20 16
-  Vendor-Specific Data Block (HDMI), OUI 00-0C-03:
-    Source physical address: 1.0.0.0
-    Maximum TMDS clock: 300 MHz
-    Supported Content Types:
-      Graphics
-    Extended HDMI video details:
-      HDMI VICs:
-        HDMI VIC 1:  3840x2160   30.000000 Hz  16:9     67.500 kHz    297.000000 MHz
-        HDMI VIC 2:  3840x2160   25.000000 Hz  16:9     56.250 kHz    297.000000 MHz
-        HDMI VIC 3:  3840x2160   24.000000 Hz  16:9     54.000 kHz    297.000000 MHz
-  Vendor-Specific Data Block (HDMI Forum), OUI C4-5D-D8:
-    Version: 1
-    Maximum TMDS Character Rate: 600 MHz
-    SCDC Present
-    Supports UHD VIC
   Video Capability Data Block:
     YCbCr quantization: Selectable (via AVI YQ)
     RGB quantization: Selectable (via AVI Q)
@@ -110,31 +93,15 @@
     IT scan behavior: Always Underscanned
     CE scan behavior: Always Underscanned
   Video Format Preference Data Block:
-    VTDB   1
-    VIC  97:  3840x2160   60.000000 Hz  16:9    135.000 kHz    594.000000 MHz
-    VIC 114:  3840x2160   48.000000 Hz  16:9    108.000 kHz    594.000000 MHz
-    DTD   1
-    DTD   3
-    DMT 0x48
-    VTDB   2
-    RID 7@60p
   YCbCr 4:2:0 Capability Map Data Block:
     VIC  96:  3840x2160   50.000000 Hz  16:9    112.500 kHz    594.000000 MHz
   YCbCr 4:2:0 Video Data Block:
     VIC 114:  3840x2160   48.000000 Hz  16:9    108.000 kHz    594.000000 MHz
   DisplayID Type VII Video Timing Data Block:
-    VTDB 1:  5120x2160   60.000000 Hz   1:1    133.320 kHz    693.264000 MHz (aspect 1:1, no 3D stereo)
-                  Hfront    8 Hsync  32 Hback   40 Hpol P
-                  Vfront   48 Vsync   8 Vback    6 Vpol N
   DisplayID Type VIII Video Timing Data Block:
-    DMT 0x48:  1920x1200  119.908612 Hz  16:10   152.404 kHz    317.000000 MHz (RB)
   DisplayID Type X Video Timing Data Block:
-    VTDB 2:  5120x1440   50.000000 Hz  32:9     73.700 kHz    383.240000 MHz (RBv3, aspect 32:9, no 3D stereo)
   Video Format Data Block:
-    RID 7@30p:  3840x1080   30.000000 Hz  32:9     33.000 kHz    134.112000 MHz
-    RID 7@60p:  3840x1080   60.000000 Hz  32:9     67.200 kHz    268.800000 MHz
   Native Video Resolution Data Block:
-    VTDB   1
   Detailed Timing Descriptors:
     DTD 2:  3840x2160   59.996625 Hz  16:9    133.312 kHz    533.250000 MHz (960 mm x 540 mm)
                  Hfront   48 Hsync  32 Hback   80 Hpol P
@@ -152,9 +119,9 @@
   Native detailed modes: 1
   Detailed Timing Descriptors:
     DTD 3:  1280x720    59.855126 Hz  16:9     44.772 kHz     74.500000 MHz (analog composite, sync-on-green, 960 mm x 540 mm)
-                 Hfront   64 Hsync 128 Hback  192 Hpol N
-                 Vfront    3 Vsync   5 Vback   20 Vpol N
-Checksum: 0xef  Unused space in Extension Block: 105 bytes
+                 Hfront   64 Hsync 128 Hback  192
+                 Vfront    3 Vsync   5 Vback   20
+Checksum: 0xef
 
 ----------------
 
