--- ref
+++ di
@@ -60,23 +60,12 @@
     VIC  18:   720x576    50.000000 Hz  16:9     31.250 kHz     27.000000 MHz
     VIC  17:   720x576    50.000000 Hz   4:3     31.250 kHz     27.000000 MHz
     VIC   1:   640x480    59.940476 Hz   4:3     31.469 kHz     25.175000 MHz
-  Vendor-Specific Data Block (HDMI), OUI 00-0C-03:
-    Source physical address: 1.0.0.0
-    DC_36bit
-    DC_30bit
-    DC_Y444
-    Maximum TMDS clock: 310 MHz
   Video Capability Data Block:
     YCbCr quantization: No Data
     RGB quantization: No Data
     PT scan behavior: Always Underscanned
     IT scan behavior: Always Underscanned
     CE scan behavior: Supports both over- and underscan
-  Vendor-Specific Data Block (AMD), OUI 00-00-1A:
-    Version: 1.1
-    Minimum Refresh Rate: 46 Hz
-    Maximum Refresh Rate: 75 Hz
-    Flags 1.x: 0xed (MCCS)
   Colorimetry Data Block:
     BT2020cYCC
     BT2020YCC
@@ -104,20 +93,12 @@
     DTD 5:  1920x1080   74.972503 Hz  16:9     83.894 kHz    174.500000 MHz (597 mm x 339 mm)
                  Hfront   48 Hsync  32 Hback   80 Hpol P
                  Vfront    3 Vsync   5 Vback   31 Vpol N
-Checksum: 0xa3  Unused space in Extension Block: 10 bytes
+Checksum: 0xa3
 
 ----------------
 
-Warnings:
-
-Block 1, CTA-861 Extension Block:
-  Video Capability Data Block: Set Selectable YCbCr Quantization to avoid interop issues.
-  Colorimetry Data Block: Set the sRGB colorimetry bit to avoid interop issues.
-  Display Product Serial Number is set, so the Serial Number in the Base EDID should be 0.
-
 Failures:
 
 Block 1, CTA-861 Extension Block:
   Video Capability Data Block: Set Selectable RGB Quantization to avoid interop issues.
-
 EDID conformity: FAIL
